### PR TITLE
Added a helm chart

### PR DESCRIPTION
**Pull request by fabian-arnold**
Thanks for creating this great application!

This PR adds a helm chart for timetagger.
GitHub workflows are included, but probably require the setup of github pages for the repository.

### CODE REVIEW

**Comment by almarklein** [comment]
Could you explain the benefit of this compared to the Docker image that's already available? Feels like running this on Kubernetes is overkill, or would this be for people who already run a Kubernetes cluster and want to add TimeTagger?

**Comment by fabian-arnold** [comment]
Yeah, it's probably overkill, but fun. I´m managing my personal applications with microk8s and created the chart for myself. As the repo already contains a docker compose file, i tought adding a helm chart may be helpfull for other k8s user's.

**Comment by almarklein** [comment]
Can we move that new charts directory to `deploy/helmchart`?

File: charts/timetagger/values.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+fullnameOverride:
+nameOverride:
+
+env:
+   - name: ""
+     value: ""
+
+secretEnv:
+   - name: TIMETAGGER_CREDENTIALS
+     value: test:$2a$08$0CD1NFiIbancwWsu3se1v.RNR/b7YeZd71yg3cZ/3whGlyU6Iny5i
```
Comment (almarklein) [inline charts/timetagger/values.yaml:36]: Should this be kept?

File: charts/timetagger/values.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+
+## TimeTagger image configuration
+##
+image:
+  repository: ghcr.io/almarklein/timetagger
+  tag: v22.9.1
```
Comment (almarklein) [inline charts/timetagger/values.yaml:6]: Does this version number here mean that we need to update this file with each release? Can we avoid/automate that?

**Review by almarklein — COMMENTED** [review]
Hi @fabian-arnold, what's the status of this? I added some comments. Is there anything else needed?

I'd be ok to add this to the repo, but only if does not cost me any additional maintenance. I don't fully understand the things that you added, so I might mention you in case other users have questions 😉 .

**Comment by almarklein** [comment]
Closing this for now. Feel free to re-open if you plan to move it forward!

**Pull request closed without merging**